contracts/MultiPartyEscrow.sol

```diff
@@ -63,8 +63,8 @@ contract MultiPartyEscrow {
     public
     returns(bool)
     {
-        require(balances[msg.sender] >= value);
-        require(token.transfer(msg.sender, value));
+        require(balances[msg.sender] >= value, "Insufficient balance in the contract.");
+        require(token.transfer(msg.sender, value), "Unable to transfer token to the contract.");
         balances[msg.sender] = balances[msg.sender].sub(value);
```

Comment:
It isn't a good practice to put the `.sub(value)` before the `.transfer()`?
I know that a reentry attack should not work here, but just in case =).

Reply:
@arturgontijo As a best practice we keep the transfer function call at the end. 
Only changes made to the MPE contract is require messages as requested.